website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "2.5.0",
```

Comment:
I think you need to regenerate the website because I modified 2 pages in a commit on which you rebased this PR...